### DISCUSSION TITLE
Title translation

### DISCUSSION THREAD

**Discussion by lucydjo** [dailydotdev/daily#1062] [Feature Requests]
![translate_button](https://github.com/dailydotdev/daily/assets/6146427/d0261a69-ef9d-49ef-8d3d-323ca703efb4)

The idea would be to be able to have a simple translation of a title thanks to a button like in the screen. It would be practical when you don't know a word (when you have intermediate English like me).

**Comment by idoshamun** [comment]
Hi @lucydjo,

Thanks for the suggestion. But the article itself is in English, do you read it via Google Translate (or similar)?

**Reply by lucydjo** [reply]
I use deepl.com when certain phrases are too complicated for me. But having a title translated quickly can let me know if the subject interests me.

**Reply by idoshamun** [reply]
Thanks for the info!
